CMakeLists.txt

```diff
@@ -405,9 +393,9 @@ endif (HAVE_SUBUNIT)
 
 ###############################################################################
 # Generate "config.h" from "cmake/config.h.in"
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/config.h.in
-  ${CMAKE_CURRENT_BINARY_DIR}/config.h)
-  # Param @ONLY not used on purpose!
+# Param @ONLY not used on purpose!
+CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/cmake/config.h.in config.h)
```

Comment:
Ah, I completely forgot the library was built as a subproject. Thanks for pointing this out, sorry about that

Reply:
Not everybody builds Check as a subproject. Many - if not most - build the Check separately and install it system wide (for all users to access). But CMake allows the inclusion of one project into another. This way of using CMake projects will probably become more popular in the future because it is easier and also more _ad hoc_ and therefore allows for quicker testing.